src/main/java/ecommerce/dao/ProductDaoImpl.java

```diff
@@ -0,0 +1,43 @@
+package ecommerce.dao;
```

Comment:
```suggestion
package ecommerce.dao.impl;
```
move all implementations to separate package

Reply:
ok